test/testdata/lsp/document_symbols.rb

```diff
@@ -1,14 +1,88 @@
 # typed: true
 
 class A
-  def foo
-    1 + 3
+  extend T::Sig
+
+  sig {void}
+  def initialize
+    @ivar = T.let(0, Integer)
+  end
+
+  sig {returns(Integer)}
+  def next
+    ret = @ivar
+    @ivar += 1
+    ret
   end
 
+  # no sig
   def self.bar
-    1 + 2
+    puts "hey there"
+  end
+end
+
+module M
+  extend T::Sig
+
+  # should not appear, since no let
+  @@nope = "doesn't work"
+  # should appear, since yes let
+  @@greeting = T.let("hello", String)
+
+  sig {params(x: String).returns(String)}
+  def hi(x)
+    "#{@@greeting}, #{x}"
+  end
+
+  sig(:final) {params(x: String).returns(String)}
+  def bye(x)
```

Comment:
Can you add a `# ^ hover` assertion here to ensure that it contains the correct sig string? (I know it's technically redundant w/ the document symbols output.)